etw/cs/traceloggingdynamic/TraceLoggingDynamic.cs

```diff
@@ -858,7 +919,7 @@ public void AddUnicodeString(string name, String value, int startIndex, int coun
         /// </summary>
         public void AddUnicodeStringArray(string name, String[] values, EventOutType outType = EventOutType.Default, int tag = 0)
         {
-            this.AddArrayMetadata(name, InType.UnicodeString, outType, tag);
+            this.AddMetadata(name, InType.UnicodeStringArray, outType, tag);

```

Comment:
Note: Since there is no longer an array-specific AddArrayMetadata, the call site is now responsible for including the Array flag in the intype.